### PR TITLE
ibm_db: Rename poolsize to poolSize

### PR DESCRIPTION
**Pull request by lasse-it**
The variable is spelled using camelCase as shown in the code [here](https://github.com/ibmdb/node-ibm_db/blob/master/lib/odbc.js#L2543).

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ibm_db ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ibm_db@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ibm_db'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 52806321eb343625c6e9ad5dcfb0162be1369ae3
</p>

**Review by MysteryBlokHed — COMMENTED** [review]
If you add a test for this change, it can be merged without needing review from a DT maintainer. You could update the test function here to include a line to test `poolSize`:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f1c7be0bf694f55a4cb8b34bd1134e3d23cf21e7/types/ibm_db/ibm_db-tests.ts#L39-L56>

Something like this under line 40 would work:

```typescript
pool.poolSize; // $ExpectType number
```

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by lasse-it** [comment]
Ready to merge

**Pull request merged**
